modules/websocket/websocket_multiplayer_peer.cpp

```diff
@@ -114,6 +114,8 @@ Error WebSocketMultiplayerPeer::get_packet(const uint8_t **r_buffer, int &r_buff
 		_current_packet.data = NULL;
 	}
 
+	ERR_FAIL_COND_V(_incoming_packets.size() == 0, ERR_DOES_NOT_EXIST);
```

Comment:
Use `ERR_UNAVAILABLE` for consistency with other PacketPeer (WebSocketPeer/PacketPeerUDP/etc).